### ISSUE TITLE
Cannot Chord Function Keys on Das Keyboard 4 Professional

### ISSUE THREAD

**Issue by JeffHanna** [Microsoft/vscode#61244]
Issue Type: <b>Bug</b>

In VSCode, both Insiders and general release, I cannot chord my function keys using a Das Keyboard 4 Professional. For instance Alt+F12 to do Peek Definition does not work but using Peek Definition from the right-click menu does work.

In the Keyboard Shortcuts dialog I can not assign a command to a chorded key combination that includes an F key. If I hold down a key, like Alt, and then try to add in an F key nothing is registered. The chording also fails in the 'Record Keyboard' feature in the search bar at the top of the Keyboard Shortcuts interface.

Disabling all extensions does not change this situation.

Chording with F keys works in every other application I have tried.

VS Code version: Code - Insiders 1.29.0-insider (9e83209ed59a22df3d593735d04cc963396bb89e, 2018-10-18T05:17:36.900Z)
OS version: Windows_NT x64 10.0.15063

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4820K CPU @ 3.70GHz (8 x 3702)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|47.94GB (35.63GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
language-maxscript|ate|1.8.3
autodocstring|njp|0.2.2
chat|kar|0.7.3
python|ms-|2018.9.0
vsliveshare|ms-|0.3.834
vscode-icons|rob|7.27.0
perforce|sle|3.1.0
vscodeintellicode|Vis|1.0.5

</details>

**Comment by sbatten** [comment]
@JeffHanna Unfortunately I am unable to repro this behavior on any release. While it doesn't change the behavior for me, does changing the setting `window.titleBarStyle` setting affect this for you? Also, does this repro with other keyboards?

@sandy081 Have you seen something like this? Any ideas?

**Comment by JeffHanna** [comment]
@sbatten & @sandy081,

It's the nVidia GeForce Experience overlay. I found this by accident by using Alt+F1 in VSCode this morning (Show accessibility options) and instead of seeing the accessibility options I instead go the GeForce Experience desktop overlay for taking a screenshot. I killed GeForce Experience and tried Alt+F12 and indeed got Peek Definition to appear.

I'm not sure if GeForce Experience can be coded around or if there is a way to have it ignore certain applications. For now I'll keep it disabled, but being a PC game developer, it is useful to have that running sometimes...
